src/fragments/lib-v1/auth/js/sms/sign_up.mdx

```diff
@@ -5,21 +5,26 @@
 
 import { signUp } from 'aws-amplify/auth';
 
-async function handleSignUp(username: string, password: string) {
-  try {
-    await signUp({
-      username,
-      password,
-      options: {
-        userAttributes: {
-          phone_number: "+18880008888",
-          email: "email@email.com",
-        },
-      },
-    });
-  } catch (error) {
-    console.log(error);
-  }
+async function handleSignUp(
+	username: string,
```

Comment:
Actually I think we need to use 2-space indentation everywhere as per the style guide: https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#21-use-2-space-indents-for-code-blocks Could you apply throughout this PR? Apologies, should have been more specific originally

Reply:
That's how prettier formatted the code.

Reply:
I'm updating the PR with the new indentation